### DISCUSSION TITLE
DInput third-party controller not working

### DISCUSSION THREAD

**Discussion by MrRumpel** [cathery/sys-con#42] [Controller requests]
![2019110918265400-DB1426D1DFD034027CECDE9C2DD914B8](https://user-images.githubusercontent.com/20477534/68527572-41267700-0323-11ea-8d4f-8a7efe980580.jpg)

**Comment by cathery** [comment]
Looks like it's trying to emulate Dualshock3/DInput. Right now it's difficult to recognize those kinds of controllers(the console doesn't show any controllers with an IClass of 3). I'll see if I can do something about it in the next few days.
